README.md

```diff
@@ -17,6 +17,11 @@ You need to copy the correct version from `WebGLTemplates/AirConsole` for Unity
 2. The content of the index.html / screen.html has changed.
 Search for `<script src="translation.js"></script>` and replace it with `<script src="airconsole-settings.js"></script>`, otherwise neither Translations nor Player Silencing will work.
 
+3. Ensure that obsolete API devices, device_id and server_time_offset are updated:
```

Comment:
Hmm from my perspective no but I might need to explain:

The obsolete attribute in C# has 2 modes - without the `, true` that I added, it shows up as compile warnings (thats what it did since that was added many years ago by Alice I think). But because Unity devs are used to seeing those, they will commonly ignore them (as most SEs with compile warnings where no CI enforces 0 warnings
If true is set, they instead become compile errors but not just meaningless ones - instead they show the text provided in the obsolete attribute so the developers have self-speaking instructions.

Due to that I would prefer to mark them with the `, true` compile at this time so any game updating to 2.5.0 gets a compile error with the instructions on how to upgrade it without first checking the upgrade notes.

Ultimately the hope would be to then remove the long obsoleted APIs finally in the next minor / major release.

Reply:
Aha that makes sense I think